tests/utils/admincli/vmgroupmgmt.go

```diff
@@ -93,13 +121,38 @@ func ConfigInit(ip string) (string, error) {
 	}
 
 	log.Printf("Initializing the SingleNode Config DB on esx [%s] \n", ip)
-	return ssh.InvokeCommand(ip, admincli.InitLocalConfigDb)
+	out, err := ssh.InvokeCommand(ip, admincli.InitLocalConfigDb)
+
+	if err != nil {
+		return out, err
+	}
+
+	// verify the DB init
+	if GetDBmode(ip) != admincli.DBSingleNode {
```

Comment:
The logic in CreateVmGroup and DeleteVmGroup makes more sense to me because we just want to add one extra verification, but don't really want to change the original successful command output.